src/content/packages.cpp

```diff
@@ -43,17 +43,14 @@ std::vector<Package> getPackagesFromURL(const std::string &url)
 	for (unsigned int i = 0; i < json.size(); ++i) {
 		Package package;
 
+		package.author = json[i]["author"].asString();
 		package.name = json[i]["name"].asString();
 		package.title = json[i]["title"].asString();
-		package.author = json[i]["author"].asString();
 		package.type = json[i]["type"].asString();
 		package.shortDesc = json[i]["shortDesc"].asString();
-		package.url = json[i]["url"].asString();
 		package.release = json[i]["release"].asInt();
-
-		Json::Value jScreenshots = json[i]["screenshots"];
-		for (unsigned int j = 0; j < jScreenshots.size(); ++j) {
-			package.screenshots.push_back(jScreenshots[j].asString());
+		if (json[i].isMember("thumbnail")) {
```

Comment:
Superfluous brackets

Reply:
Removing this could cause an else ambiguity